unit-tests/src/test/java/org/eclipse/collections/impl/bag/immutable/ImmutableEmptyBagTest.java

```diff
@@ -576,7 +574,6 @@ public void groupByUniqueKey()
     @Test
     public void groupByUniqueKey_throws()
     {
-        super.groupByUniqueKey_throws();
```

Comment:
I'm curious about these removals of the calls to super.

Reply:
The super class call didn't do an explicit assertion for an exception due to the @Override which leads to the old (expected = xx.class) being ignored.

Now that we have moved to the junit5 style, we have explicit assertion that fails the test in the super.() call.